t/fcodes_s.t

```diff
@@ -215,7 +215,7 @@ is(
 # Test HTML output of links.
 use Pod::Simple::HTML;
 my $PERLDOC = "https://metacpan.org/pod";
-my $MANURL = "http://man.he.net/man";
+my $MANURL = "https://man7.org/linux/man-pages";
```

Comment:
This test fails (the new URL suffix is missing here), but both the generated and expected URLs are wrong (404).

Reply:
Introduced `$MANURL_POSTFIX` in tests.